### ISSUE TITLE
[Eclipse-ThirdParty][A11yMAS][Accessibility][Narrator][MAS4.2.12][Eclipse][Wipro]:Narrator is not announcing the different error information for invalid entries in "Work with" combo box

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1418]
**Repro Steps:**

1. Open and Start Eclipse.
2. Navigate to Help menu.
3. Navigate and select Install new software Menu item under Help menu.
4. Install dialog will open.
5. In "Install" dialog , tab to Work with text box, and gave any invalid data.
6. Observe the narrator behaviour

**Actual Result:**
Narrator and NVDA is not announcing the different  error informations of the combo box

**Narrator Behaviour:**
Tab
Work with:, editable edit, Alt+ w, 

**NVDA Behaviour:**
Work with:  combo box  collapsed
edit  Alt+w

**Expected Result:**
Narrator and NVDA should announce the different error informations for the combo box

**Platform:**
Eclipse Version:
Eclipse Java EE IDE for Web Developers.
Version: Oxygen.2 Release (4.7.2)
Build id: 20171218-0600
OSBuild Version: Version 1709(Os Build 16299.192)
![available software-error-acc event](https://user-images.githubusercontent.com/22956130/35717496-850bb07c-0805-11e8-8149-25424277f742.jpg)
![error-inspect](https://user-images.githubusercontent.com/22956130/35717513-ada11cac-0805-11e8-9b3f-03ebe091f3d6.jpg)
![error-narrator](https://user-images.githubusercontent.com/22956130/35717514-b0b4a7b0-0805-11e8-9d46-db8e1ba2c437.jpg)
![error-nvda](https://user-images.githubusercontent.com/22956130/35717518-b35e6bf4-0805-11e8-9eae-8caea6ab1c86.jpg)

**Comment by Eskibear** [comment]
It is determined by Eclipse IDE. We cannot fix it.

**Comment by Eskibear** [comment]
And it's out of the toolkit's scope.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749870
